### ISSUE TITLE
Documentation is obsolete: ldap configuration

### ISSUE THREAD

**Issue by mrPsycho** [opnsense/core#3055]
Hello,

while i was configuring new OPNsense 18.7.9 - i noticed that https://wiki.opnsense.org/manual/how-tos/user-ldap.html is obsolete. 

for example, import users button didn't show up until i set "Authentication" server to LDAP. 
also, i didn't found any option to give permission to LDAP group.

**Comment by mrPsycho** [comment]
actually, almost all other topics are old too. 

another example - https://wiki.opnsense.org/manual/how-tos/haproxy.html
which is confusing...

**Comment by mimugmail** [comment]
Yep, progress in coding is faster than docs, but luckily everyone can contribute a bit time to update it :)

**Comment by AdSchellevis** [comment]
If you would like to help improving the documentation, that would certainly be appreciated.

You can find the repository here https://github.com/opnsense/docs
